bodhi/server/bugs.py

```diff
@@ -201,15 +201,17 @@ def close(self, bug_id, versions, comment):
                 fixedin = [v.strip() for v in bug.fixed_in.split()]
                 # Strip out any empty strings (already stripped)
                 fixedin = [v for v in fixedin if v]
-                # And add our build if its not already there
-                if version not in fixedin:
-                    fixedin.append(version)
 
                 # There are Red Hat preferences to how this field should be
                 # structured.  We should use:
                 # - the full NVR as it appears in koji
                 # - space-separated if there's more than one.
-                args['fixedin'] = " ".join(fixedin)
+                fixedinString = " ".join(fixedin)
+                
+                # Add our build if its not already there
+                # but only if resultant string length is lower than 256 chars
+                if (version not in fixedin) and (len(fixedinString)+len(version) < 255):
```

Comment:
Also for [PEP-8](https://www.python.org/dev/peps/pep-0008/#other-recommendations), there should be spaces surrounding the ```+``` operator.

Reply:
```flake8``` mentioned this too in the CI test run:

```
rawhide	./bodhi/server/bugs.py:213:68: E226 missing whitespace around arithmetic operator
rawhide	                if (version not in fixedin) and (len(fixedinString)+len(version) < 255):
rawhide	     
```